jarviscli/plugins/teachme.py

```diff
@@ -0,0 +1,420 @@
+from nltk.tokenize import word_tokenize as wt, sent_tokenize as st
+from nltk.corpus import wordnet
+from nltk.stem import WordNetLemmatizer
+from collections import Counter
+import nltk
+import numpy as np
+import matplotlib.pyplot as plt
+from nltk.corpus import stopwords
+from nltk import FreqDist
+import os
+import requests
+import json
+import spacy
+import random
+import requests
+from google_trans_new import google_translator
+from difflib import SequenceMatcher
+
+from plugin import plugin
+
+FILE_PATH = os.path.abspath(os.path.dirname(__file__))
+
+nltk.download('punkt')

```

Comment:
Should be done in ``installer/steps/c_nltk.py``

(simply add stopwords to package -> don't include ``jarviscli/data/4300fullstopwords.txt`` in git

Reply:
Ok, I changed it, we didn't noticed that there were a special step for nltk.